### DISCUSSION TITLE
Qdrant hook

### DISCUSSION THREAD

**Discussion by raulcarlomagno** [apache/airflow#25902] [General]
### Description

Having a Qdrant hook should make things easier when working with it.

### Use case/motivation

Use a Qdrant hook, as i do with elasticsearch or mongodb, helps to organize and make code cleaners

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
We had a long discussion on devlist: https://lists.apache.org/thread/6ngq79df7op541gfwntspdtsvzlv1cr6 about accepting new providers and I think you should rather consider to follow the path of Great Expectations and others and release your own provider on your own.

There are certain requirements and obligations for a new provider to be accepted in community https://github.com/apache/airflow/blob/main/README.md#release-process-for-providers.

Unless your service is very popular already, we are rather reluctant to approve new integrations with smaller and less popular services, instead we encourate people to publish their own providers on PyPI on their own:

https://airflow.apache.org/docs/apache-airflow-providers/index.html#how-to-create-your-own-provider

There are ways you can publish information about the new providers that you release :

https://airflow.apache.org/ecosystem/
https://registry.astronomer.io/
And likely a few other places.

**Comment by potiuk** [comment]
Converted it into discussion in case more discussion is needed.

**Comment by Anush008** [comment]
Qdrant provider is now available.

https://airflow.apache.org/docs/apache-airflow-providers-qdrant/stable/index.html

**Reply by raulcarlomagno** [reply]
👏 thank you!
